archai/nlp/nvidia_transformer_xl/nvidia_utils/proj_adaptive_softmax.py

```diff
@@ -151,6 +154,11 @@ def default_tie_proj(cutoffs:List[int])->List[bool]:
     def clean_cutoffs(cutoffs:Optional[List[int]], n_token:int):
         if cutoffs is None:
             cutoffs = ProjectedAdaptiveLogSoftmax.default_cutoffs(n_token)
+
+        # Not using adaptive emb
+        if len(cutoffs) == 0:
```

Comment:
Why do we need this? It looks to me that empty cutoffs are handled properly. Instead of premature return, we return proper cutoff if cutoff was empty array.